### DISCUSSION TITLE
将自动化服务变成你的导出服务

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2151] [服务]
> **[简悦 2.2.0](http://ksria.com/simpread/welcome/version_2.2.0.html)** 发布后，增加了自动化功能，通过此功能，可以联动多个服务，你可以理解为 **简悦的 IFTTT**。

> 自动化功能是 2.2.0 最重要的一个更新，建议动手能力强的用户，直接 [查看文档](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%8A%A8%E5%8C%96)

## 配置

> 将 **稍后读** 改为 **手动模式** 后，就可以成为 **导出服务**，因此会跟其它导出服务一样，在需要时点击使用。

![image](https://user-images.githubusercontent.com/81074/119219214-f8a26900-bb16-11eb-8ef0-14cb089d4200.png)

![image](https://user-images.githubusercontent.com/81074/119219202-e1fc1200-bb16-11eb-92cd-1a7e955d656f.png)

## 使用

[![g7Gz2n.png](https://z3.ax1x.com/2021/05/21/g7Gz2n.png)](https://imgtu.com/i/g7Gz2n)

[![g7J3Ie.png](https://z3.ax1x.com/2021/05/21/g7J3Ie.png)](https://imgtu.com/i/g7J3Ie)

## 一个例子

![2021-05-22_16-04-15 (1)](https://user-images.githubusercontent.com/81074/119219333-9d24ab00-bb17-11eb-9773-ee363b210a30.gif)

**Comment by ajiangstudy** [comment]
您好，我通过自动化无法自动同步到notion上。选择手动或者批注都可以成功同步到notion，请问是什么问题？

**Reply by Kenshin** [reply]
嗨，看下这个说明 https://github.com/Kenshin/simpread/discussions/2362
